SPECS/node-problem-detector/node-problem-detector.spec

```diff
@@ -61,6 +62,10 @@ make test
 %config(noreplace) %{_sysconfdir}/node-problem-detector.d/*
 
 %changelog
+* Fri Feb 25 2022 Max Brodeur-Urbas <maxbr@microsoft.com> - 0.8.10-1
+- Upgrading to v0.8.10
+- Disable arm64 builds in Makefile with remove_windows_build.patch.
```

Comment:
Please create a separate patch - the name now no longer reflects what the patch is doing.

Reply:
Alternatively, you can also re-name the patch file but it's cleaner to keep non-related changes apart.